content/css/concepts/floats/floats.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'Floats'
+Description: 'The float property moves an element to the left or right within its parent, outside of the default positioning.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Floats'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `float` property moves an element to the left or right within its containing element, outside of the default [positioning](https://www.codecademy.com/resources/docs/css/position).
+
+## Syntax
+
+The `float` property primarily uses the following three values:
+
+* The `left` value that makes the element float on the left of its container.
+* The `right` value that makes the element float on the right of its container.
+* The `none` value that causes the element not to float.
+
+```pseudo
+.item_x {
+  float: none;
+}
+
+.item_y {
+  float: left;
+}
+
+.item_z {
+  float: right;
+}
+```
+
+## Example
+
+```html
+<section>
+  <div class="item_1">
+    <p>This is the first item and I will float left.</p>
+  </div>
+  <div class="item_2">
+    <p>This is the second item and I will float right</p>
+  </div>
+    <br>
+    <br>
+    <br>
+  <p>This example shows CSS floats and I wont float at all.</p>
+</section>
+```
+
+In the above snipit the elements with the class `item_1` and `item_2` you can set the float in an external style sheet.
+
+```css
+.item_1 {
+  float: left;
+}
+
+.item_2 {
+  float: right;
+}
+
+section > p {
+  border: thick double black;
+}
```

Comment:
Actually let's keep that `<div>` rule for the borders and add the following additional rule:

```suggestion
div {
  border: thick double black;
}

section > p {
  margin: auto;
  text-align: center;
  width: 50%;
}
```